modules/sdk-coin-sol/src/lib/tokenTransferBuilder.ts

```diff
@@ -1,21 +1,15 @@
-import { BaseCoin as CoinConfig, coins, SolCoin } from '@bitgo/statics';
+import { BaseCoin as CoinConfig, SolCoin } from '@bitgo/statics';
 import { BuildTransactionError, TransactionType } from '@bitgo/sdk-core';
 import { Transaction } from './transaction';
-import { getAssociatedTokenAccountAddress, isValidAmount, validateAddress } from './utils';
+import { getAssociatedTokenAccountAddress, getSolTokenFromTokenName, isValidAmount, validateAddress } from './utils';
 import { InstructionBuilderTypes } from './constants';
-import { TokenTransfer } from './iface';
+import { TokenTransfer, Recipient } from './iface';
 
 import assert from 'assert';
 import { TransactionBuilder } from './transactionBuilder';
 
-export interface SendParams {
```

Comment:
Undocumented breaking change here.

Reply:
Ohh I only move the interface to `iface.ts`, I'll just undo it since it's not involved in the main change

Reply:
Thanks for pointing to that. In that case, it wouldn't be a breaking change, since we don't support `import from ... dist/` and it's all being exported via `*` from lib. Carry on :)